modules/end-user-guide/pages/supported-languages.adoc

```diff
@@ -1,10 +1,18 @@
 :_content-type: REFERENCE
-:description: Supported languages
+:description: Stack samples
 :keywords: user-guide, languages
-:navtitle: Supported languages
+:navtitle: Stack samples
 :page-aliases:
 
 [id="supported-languages"]
-= Supported languages
+= Stack samples
 
 include::example$snip_{project-context}-supported-languages.adoc[leveloffset=+1]
+
+[NOTE]
+====
+The amount of supported languages may vary depending on the selected IDE. For a full list of languages supported by your IDE, refer to its documentation.
+====
+
+.Additional resources
+. link:https://github.com/eclipse-che/che-devfile-registry/tree/main/devfiles[Git repository] of stack samples using various programing languages.
```

Comment:
I suggest you do another `include` statement here:
```suggestion
include::example$snip_{project-context}-link-to-devfile-registry.adoc[]
```
and move this bullet point to a snip file in the examples folder with just a line in it like this:
```
* link:https://github.com/eclipse-che/che-devfile-registry/tree/main/devfiles[Devfiles of {prod-short} stack samples on GitHub]
```
and then add a downstream link as well.